libs/auth/src/user-repository/index.ts

```diff
@@ -0,0 +1,66 @@
+import { prisma } from "@hmcts/postgres";
```

Comment:
I thought in one of your previous tickets, you create a lib called account? Can it be moved there. In account create a folder called repository and add the repository method in a class called queries.ts. Please refer to publication/location and follow the same style